composer.json

```diff
@@ -29,8 +32,8 @@
             "email": "matthew@kilgore.dev"
         },
         {
-          "name": "Alexander (Sasha) Skvortsov",
-          "email": "askvortsov@flarum.org"
+            "name": "Alexander (Sasha) Skvortsov",
```

Comment:
These changes should be reverted, since individual names were removed from this file recently.

Reply:
rebased